### ISSUE TITLE
Wild bug appears!  Please copy this text and open an issue on GitHub. Node Launcher version 6.0.5  Error information:  'utf-8' codec can't decode byte 0xe9 in position 53: invalid continuation byte File "node_launcher\gui\system_tray_widgets\output_widget.py", line 24, in handle_output 

### ISSUE THREAD

**Issue by PaulADW** [lightning-power-users/node-launcher#334]
* **I'm submitting a ...**
  - [x] bug report
  - [ ] feature request
  - [ ] support request

* **What is the current behavior?**

Node launcher crashed and told me to post here so I created an account in order to help. 

* **If the current behavior is a bug, please provide the steps to reproduce**

Node launcher was roughly 60% synchronized with the bitcoin network but crashed.

* **What is the expected behavior?**

Sync fully.

* **What is the motivation / use case for changing the behavior?**

Being able to sync.

* **Please tell us about your environment:**

  - Operating System: [ Windows]

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, gitter, etc)

Here's what it told me to post : 

Wild bug appears! 
Please copy this text and open an issue on GitHub.
Node Launcher version 6.0.5

Error information:

'utf-8' codec can't decode byte 0xe9 in position 53: invalid continuation byte
File "node_launcher\gui\system_tray_widgets\output_widget.py", line 24, in handle_output

**Comment by PaulADW** [comment]
Oh it made some more errors : 


Internal C++ object (PySide2.QtCore.QProcess) already deleted

**Comment by PierreRochard** [comment]
Thank you for the bug report! Are you using English as the Windows system language?

**Comment by PaulADW** [comment]
No my system is set up in French, do you want me to try relaunching the
whole thing with english as my system's language ? I could do that if it
helps you identify the problem.

Le jeu. 2 mai 2019 à 04:48, Pierre Rochard <notifications@github.com> a
écrit :

**Comment by PierreRochard** [comment]
Yes, that would be helpful, as I've had it solve the problem for one person already. If it solves it for two people, then I can address it in the code as I do want the Node Launcher to be compatible with every language (and eventually translate it itself)

**Comment by editor-Ajian** [comment]
Here I meet the similar problem:

Error information:

'utf-8' codec can't decode byte 0xb3 in position 41: invalid start byte
File "node_launcher\gui\system_tray_widgets\output_widget.py", line 24, in handle_output

**Comment by PaulADW** [comment]
Hey guys !

Just to let you know that switching my system's language to english also
solved the problem for me.
Now next problem I have : I've seen the tutorial to use Zap Desktop to send
payments, etc.

How do I make the Zap iOS app work with node launcher ? The QR Code I get
on Zap Desktop links to a localhost, which means that Zap iOS can't reach
it.
Also, is there a mnemomic phrase for my lightning node if such a thing
exists ? Or does it rely solely on the bitcoin wallet linked to it ?

Sorry for noobish questions, i'm Learning everyday.
Paul





Le lun. 6 mai 2019 à 06:35, editor-Ajian <notifications@github.com> a
écrit :
